### PR TITLE
Add credentials/employee

### PR DESCRIPTION
**Pull request by halvey58078**
**modAddCred.py:** adds credentials for employees into the Employees table
**modInterface.py:** added buttons for adding employee/credentials, added button for updating product, changed button size and location for add product and show expiration date
**Queries.py:** added query to add employee credentials

### CODE REVIEW

File: Queries.py [hunk 1]
```diff
@@ -80,3 +80,22 @@ def addProductQuery(self, mycursor, productList, connection):
         mycursor.execute(sql_inventory_insert, sql_inventory_val)
 
         connection.saveData()
+
+
+    @classmethod
+    def addCred(self, mycursor, empInfo, connection):
+        sqlvar1 = "insert into employees \
+                  (employees.emp_num, employees.f_name, employees.m_init, employees.l_name, employees.dept_num, employees.cridencials)\
```
Comment (heidtna) [inline Queries.py:88]: "credentials".

File: modInterface.py [hunk 2]
```diff
@@ -2,6 +2,8 @@
 from modConnection import *
 import Queries
 from modAddProduct import *
+from modAddCred import *
+from modUpdateProd import *
```
Comment (heidtna) [inline modInterface.py]: I didn't see the addition of the 'modUpdateProd' file anywhere. Did it just not get included in the commit?

**Comment by heidtna** [comment]
Please change the variable name for the first button to be "self.btnUpdateProduct"
image

**Comment by halvey58078** [comment]
this became a cluster mess. closing due to chaos.

**Pull request closed without merging**